releasenotes/notes/stretch-duration-alignments-cd8395bcc8f6cdd0.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Circuits containing delays with stretches (see :meth:`.QuantumCircuit.add_stretch`) can now
+    successfully compile using the preset pass-managers (:func:`.transpile` and
+    :func:`.generate_preset_pass_manager`) when targetting a backend that has alignment constraints,
+    for example IBM Quantum Eagle devices like ``ibm_washington``.
```

Comment:
Oh whoops, I meant `ibm_sherbrooke` - I think Washington got retired a few months ago?

Reply:
`ibm_washington` was retired in June 2023: https://docs.quantum.ibm.com/guides/retired-qpus